nano/node/websocket.cpp

```diff
@@ -4,6 +4,45 @@
 #include <nano/node/node.hpp>
 #include <nano/node/websocket.hpp>
 
+nano::websocket::confirmation_options::confirmation_options (boost::property_tree::ptree const & options_a) :
+all_local_accounts (options_a.get<bool> ("all_local_accounts", false))
+{
+	auto accounts_l (options_a.get_child_optional ("accounts"));
+	if (accounts_l)
+	{
+		for (auto account : *accounts_l)
+		{
+			// Check if the account is valid, but no error handling if it's not, simply not added to the filter
+			nano::account result (0);
+			if (!result.decode_account (account.second.data ()))
+			{
+				accounts.insert (account.second.data ());
+			}
+		}
+	}
+}
+
+bool nano::websocket::confirmation_options::filter (boost::property_tree::ptree const & message_a, nano::node & node_a) const
+{
+	// If this fails, the message builder has been changed
+	auto account_text (message_a.get<std::string> ("message.account"));
```

Comment:
Should this also check `message.block.link_as_account` aka destination?

Reply:
We reached the conclusion that it makes sense to and not use multiple options e.g. destination_only, source_and_destination etc to keep it simple; services can filter on their end.

Reply:
@cryptocode is it worth it to check subtype and consequently only source/destination depending on the block type?

Reply:
Right now it will get whatever block mentions the given account, be it epoch, change (source or delegation), send and receive. I think it's better to be inclusive.